### ISSUE TITLE
Document and write a README, wiki

### ISSUE THREAD

**Issue by pakoito** [asco33/rxnearby#1]
The library looks neat, but needs some love before we consider using it.

**Comment by asco33** [comment]
Yeah, it's currently at the 'idea' stage. I'll be providing a little how-to soon and improve on functionality :)

**Comment by asco33** [comment]
Wrote a very basic documentation.
